api/proto/teleport/legacy/types/types.proto

```diff
@@ -5478,6 +5487,12 @@ message SAMLAuthRequest {
   teleport.attestation.v1.AttestationStatement ssh_attestation_statement = 21 [(gogoproto.jsontag) = "ssh_attestation_statement,omitempty"];
   // TlsAttestationStatement is an attestation statement for the given TLS public key.
   teleport.attestation.v1.AttestationStatement tls_attestation_statement = 22 [(gogoproto.jsontag) = "tls_attestation_statement,omitempty"];
+  // PostForm is the HTML form value that contains the SAML authentication request data.
+  // Value is only set if the PreferredRequestBinding in the SAMLConnectorSpecV2
+  // is "http-post". In any other case, RedirectURL field will be populated.
+  bytes PostForm = 23 [(gogoproto.jsontag) = "post_form,omitempty"];
+  // ClientVersion is the version of tsh or Proxy that is sending the SAMLAuthRequest request.
+  string ClientVersion = 24 [(gogoproto.jsontag) = "client_version,omitempty"];
```

Comment:
This PR strictly does not need a `ClientVersion` value. 
In the auth, all I am [checking](https://github.com/gravitational/teleport.e/pull/6493/files#diff-7cf8f553f23a880c5668f0e748dc216694c1419db076bb2f00fde40aea9007dfR184) is if the value is provided or is empty, which a boolean value can satisfy.
But I believe `ClientVersion` field will come in handy for other use cases too.